### PR TITLE
Softdil/8.0 sale commission report qweb settlement condensed

### PR DESCRIPTION
**Pull request by hendrixcosta**
depends  #91

### CODE REVIEW

**Comment by elicoidal** [comment]
Can you please resolve the conflicts first?

File: sale_commission/report/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline sale_commission/report/__init__.py:1]: @Hendrixcosta   Thanks for your contribution to the OCA. 
Let me introduce to you some basic information about contribution in OCA and I will review other points in more comments inside the code..
In general, please follow the principles: 
- Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
- We recommend to include a screenshot as well to help new comers (nice to have)
- Please use simple headers in all py files following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: sale_commission/report/sale_commission_analysis_report.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        'Invoice line',
+        readonly=True)
+    settled = fields.Boolean('Settled', readonly=True)
+    commission_id = fields.Many2one(
+        'sale.commission',
+        'Sale commission',
+        readonly=True)
+
+    def _select(self):
+        select_str = """
```
Comment (elicoidal) [inline sale_commission/report/sale_commission_analysis_report.py:64]: I do not see anything about company information and security rules. Would it be wise to have it?

Comment (aitorbouzas) [inline sale_commission/report/sale_commission_analysis_report.py:64]: @elicoidal do you mean multi-company? There's already a PR for that in #75

Comment (elicoidal) [inline sale_commission/report/sale_commission_analysis_report.py:64]: @Softdil This is what I mean and for what I can see the PR#75 solves the multi-company scope but you current report allows to see everything which is dangerous and somehow incompatible with the other module.
I would suggest to include this report as well in the #75 to override this report and properly take into account the multi-company feature.
What do you think?

Comment (aitorbouzas) [inline sale_commission/report/sale_commission_analysis_report.py:64]: @elicoidal Ok, it would be nice, how should we proceed?

Comment (elicoidal) [inline sale_commission/report/sale_commission_analysis_report.py:64]: This PR is OK for the report (as there is no company scope).
You should propose a PR on the repo hosting #75 adding the report in the multi-company module in order to override this one (please check the details as I am not specially tech guy).

**Comment by pedrobaeza** [comment]
Is this going to be fixed?

**Comment by pedrobaeza** [comment]
Closing as no answer

**Pull request closed without merging**